### ISSUE TITLE
mime type for auth is broken

### ISSUE THREAD

**Issue by hunabku94** [shiftkey/desktop#951]
### The problem

mimtype is broken because the 3.3.3 use `x-github-desktop-dev-auth` instead of `x-github-desktop-auth`

### Release version

3.3.3

### Operating system

Archlinux

**Comment by eylenburg** [comment]
That fixed it! I associated Github Desktop with these two mimetypes in the KDE Settings -> Applications -> File Associations, and now I can log in!

**Comment by shiftkey** [comment]
@eylenburg @hunabku94 I don't publish the Arch Linux releases, but I know they use this packaging so perhaps something was changed that caused the mimetypes to be broken recently?

**Comment by eylenburg** [comment]
I use the Flatpak and had this issue

**Comment by amarnadh-18** [comment]
Hello, @eylenburg could tell how to add the mime type in  the applications i am also facing the same issue because of flatpak.

**Comment by eylenburg** [comment]
On KDE you can do it like this: KDE Settings -> Applications -> File Associations -> select "application" -> Add -> enter "x-github-desktop-auth", then choose Github Desktop as the associated application

I'm not sure how to do it on other desktop environments

**Comment by amarnadh-18** [comment]
Thank you for the quick response @eylenburg but this doesn't seem to be working in Endeavouros for some reason

**Comment by VoxelCubes** [comment]
I spent a good 2 hours bashing my head against the screen but this finally worked:
- I added the file association in system settings for x-github-desktop-auth.
- After doing the browser login thing, the github page would sit there and provide me with a link to try again manually. Copying the link's address showed me this (random numbers omitted): x-github-desktop-dev-auth://oauth?code=xxx&state=xxx
- I just removed the -dev from the url there and pasted it back into my browser. Then it just worked and everything is once again as it should be.

Seems like a simple fix.
